tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "outDir": "dist",
     "target": "es2018",
-    "lib": ["es2018"],
+    "lib": ["es2018", "DOM"],
```

Comment:
Oh I didn't notice this before. Is `DOM` necessary for something?

Reply:
For the vis types

Reply:
ahh okay